src/olm/account/mod.rs

```diff
@@ -690,6 +694,49 @@ mod test {
 
     const PICKLE_KEY: [u8; 32] = [0u8; 32];
 
+    #[test]
+    fn max_number_of_one_time_keys_matches_global_constant() {
+        assert_eq!(Account::new().max_number_of_one_time_keys(), PUBLIC_MAX_ONE_TIME_KEYS);
```

Comment:
This one is slightly silly but I'm not sure how else to fix this miss:

```
MISSED   src/olm/account/mod.rs:152:9: replace Account::max_number_of_one_time_keys -> usize with 0 in 0.8s build + 2.9s test
```

Reply:
Yeah, I think that's fine, it's a simple getter, those will always require kinda dumb tests.